src/main/java/picard/analysis/CollectWgsMetricsWithNonZeroCoverage.java

```diff
@@ -196,18 +198,9 @@ public void addToMetricsFile(final MetricsFile<WgsMetrics, Integer> file,
             }
         }
 
-        private Histogram<Integer> depthHistogramNonZero() {
-            final Histogram<Integer> depthHistogram = new Histogram<>("coverage", "count");
-            // do not include the zero-coverage bin
-            for (int i = 1; i < depthHistogramArray.length; ++i) {
-                depthHistogram.increment(i, depthHistogramArray[i]);
-            }
-            return depthHistogram;
-        }
-
-        public boolean areHistogramsEmpty() {
-            return (null == depthHistogram || depthHistogram.isEmpty());
-        }
+//        public boolean areHistogramsEmpty() {
```

Comment:
Will bring back all of the plotting related code when I resolve conflicts with Nils's changes.